src/object/schedcontext.c

```diff
@@ -172,7 +172,6 @@ static exception_t invokeSchedContext_YieldTo(sched_context_t *sc, word_t *buffe
 
     bool_t return_now = true;
     if (isSchedulable(sc->scTcb)) {
-        refill_unblock_check(sc);
```

Comment:
Does this need to be added back with !sc_sporadic(tcb->tcbSchedContext) around it?

Reply:
I don't think so. If the SC is sporadic then we don't unblock it as this isn't actually an activation and if the SC is continuous bandwidth limited then the unblock will happen in the switch to anyway.

Reply:
This was redundant for non sporadic and conflicted with sporadic (which is why it was not problematic before this PR)